generics/constants/endpoints.js

```diff
@@ -50,6 +50,7 @@ module.exports = {
 	GET_LOCATION_DATA: '/v1/location/search',
 	CERTIFICATE_CREATE: '/api/v1/ProjectCertificate',
 	PROJECT_CERTIFICATE_API_CALLBACK: '/v1/userProjects/certificateCallback',
+	PROJECT_CERTIFICATE_API_CALLBACK_ERROR: '/v1/userProjects/certificateCallbackError',
```

Comment:
@Prajwal17Tunerlabs what is the use of this endpoint?

Reply:
certificateCallbackError endpoint handles when Gotenberg throws error or exception , as per the Gotenberg documentation this is how it is done, below is the reference
https://gotenberg.dev/docs/webhook